local-dev/util/Dockerfile.server-quick

```diff
@@ -0,0 +1,9 @@
+FROM amazoncorretto:17
```

Comment:
we can encapsulate this inside the same script:

example:

 docker build -f ${SCRIPT_DIR}/util/Dockerfile.server-quick \
        --tag littlehorse/littlehorse-server:lates -<<EOF
WORKDIR /lh

COPY ./docker/server/docker-entrypoint.sh /lh
COPY ./docker/server/log4j2.properties /lh
COPY ./server/build/libs/server-*-all.jar /lh/server.jar
ENTRYPOINT ["/lh/docker-entrypoint.sh"]
CMD ["server"]
EOF

Reply:
it's just a suggestion hehe

Reply:
Roger Roger. Making that change then merging.

Reply:
I got an error after playing around with it for a few minutes, and I don't see the option here: https://docs.docker.com/engine/reference/commandline/build/

I'm going to merge as-is, and we can improve it later.